runtime/src/consensus/registry.rs

```diff
@@ -59,8 +59,8 @@ pub struct TLSInfo {
     /// Public key that will be used for establishing connections after certificate rotation (if enabled).
     pub next_pub_key: PublicKey,
 
-    /// List of addresses at which the node can be reached.
-    pub addresses: Option<Vec<TLSAddress>>,
+    #[cbor(rename = "addresses", optional)]
+    pub _deprecated_addresses: Option<Vec<TLSAddress>>,
```

Comment:
I guess v3 descriptors should deserialize fine with v2 because the addresses field will just be set to `None`?

Reply:
Yeah.  This is/should be also  covered with tests, e.g.: 
```
qWF2A2JpZFggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABjcDJwomJpZFggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABpYWRkcmVzc2Vz9mN0bHOiZ3B1Yl9rZXlYIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAbG5leHRfcHViX2tleVggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABlcm9sZXMAaHJ1bnRpbWVz9mljb25zZW5zdXOiYmlkWCAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAGlhZGRyZXNzZXP2aWVudGl0eV9pZFggAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABqZXhwaXJhdGlvbgA
```
is one of the V3 descriptors that is tested to deserialize correctly (both on go and rust sides).

`test_consistent_node` tests various V3 descriptors to deserialize correctly.